tests/consequence_prediction/vep_mapping_pipeline/resources/output_mappings.tsv

```diff
@@ -832,7 +832,8 @@
 19:40396353:G:C	1	ENSG00000105227	PRX	missense_variant	0
 19:40757356:T:C	1	ENSG00000077312	SNRPA	missense_variant	0
 19:4101085:G:A	1	ENSG00000126934	MAP2K2	synonymous_variant	0
-19:41410644:C:A	1	ENSG00000248098	BCKDHA	splice_polypyrimidine_tract_variant	0
+19:41410644:C:A	1	ENSG00000248098	BCKDHA	missense_variant	0
+19:41410644:C:A	1	ENSG00000255730		missense_variant	0
```

Comment:
Most of the changes in this file are due to the slight reordering of the newer Ensembl terms to be consistent with Ensembl's ranking API; this seems to be the only difference from the recent Ensembl 106 update.